src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisAnonymousEmpty.java

```diff
@@ -47,4 +47,18 @@ public void fooEmpty() {
             }
         };
     }
+
+    public void anotherMethod() {
+
+        this.anonMethod(new AnonWithEmpty() {
+            private int member;
+
+            @Override
```

Comment:
`@Override` this annotations is not needed in test inputs

Reply:
Done.

Reply:
still see it here :)

Reply:
Done. (for real this time)